### DISCUSSION TITLE
Release v1.15.1 / 5.70.1

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#4341] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### Release Notes

This build fixes a couple issues with 1.15.0 and updates compatibility to the latest windows insider build.

For a full list of changes please review the [change log](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1151--5701---2024-10-29).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.15.1'>Release v1.15.1 / 5.70.1</a>.</em>

**Comment by githubusername2024** [comment]
---------------------------
Error
---------------------------
Unable to execute file in the temporary directory. Setup aborted.



Error 225: Operation did not complete successfully because the file contains a virus or potentially unwanted software.
---------------------------
OK   
---------------------------

W11 24H2

![image](https://github.com/user-attachments/assets/338cdfae-9ac9-4e0a-80ca-ba51cadf41b4)

**Reply by offhub** [reply]
https://github.com/sandboxie-plus/Sandboxie/labels/False%20Positive, see https://github.com/sandboxie-plus/Sandboxie/issues/2643.

**Reply by githubusername2024** [reply]
So, do I need to just sign the *.tmp file to resolve the issue?

**Reply by offhub** [reply]
No, you need to temporarily disable the antivirus real-time protection and then rerun the installer, as the file always extracts to a different temporary folder.

**Reply by plofhaan** [reply]
> So, do I need to just sign the *.tmp file to resolve the issue?

Windows Security (Defender) lets you use wildcards to specify the filepath of the file. See: https://learn.microsoft.com/en-us/defender-endpoint/configure-extension-file-exclusions-microsoft-defender-antivirus#use-wildcards-in-the-file-name-and-folder-path-or-extension-exclusion-lists
In an elevated PowerShell window, you can run the command below to effectively eliminate both the problem of the file always extracting to a different temporary folder and the problem of the file having a different version in its filename each time when you choose to install a different version of Sandboxie-Plus x64.
`Add-MpPreference -ExclusionPath "$env:LocalAppData\Temp\*.tmp\Sandboxie-Plus-x64-v*.tmp"`

**Comment by Ailbeenlynd** [comment]
Is EnableEFS=y for subscribers only? It doesn't work in sbie classic.

**Reply by offhub** [reply]
Yes, the `EnableEFS` setting requires an **advanced** supporter certificate upgrade.

**Reply by Ailbeenlynd** [reply]
Maybe in changelogs in the future, indicate when a feature is for paid subscribers only?

**Reply by isaak654** [reply]
2674c9a

**Comment by acidhorse1972** [comment]
Congratulations, Mozilla identifies Release v1.15.1 / 5.70.1 as a virus or malware
McAfee has taken over Mozilla
![Screenshot 2024-11-18 221746](https://github.com/user-attachments/assets/e2f701f0-a48b-45a6-a12f-ed301568c45b)

**Reply by isaak654** [reply]
A new minor release is available: https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.15.4

See also:
1. https://github.com/sandboxie-plus/Sandboxie/issues/4343
2. https://github.com/sandboxie-plus/Sandboxie/issues/2646#issuecomment-1407468786
